index.html

```diff
@@ -379,8 +379,13 @@ <h2 id="hidden-display-surfaces">
           A <a>display surface</a> that is being shared may temporarily or permanently become
           inaccessible to the application because of actions taken by the operating system or user
           agent. What makes a <a>display surface</a> considered inaccesible is outside the scope
-          of this specification, but examples MAY include a <a>monitor</a> disconnecting or an
-          <a>application</a>, <a>window</a> or <a>browser</a> closing or becoming minimized.
+          of this specification, but examples MAY include a <a>monitor</a> disconnecting, an
+          <a>application</a>, <a>window</a> or <a>browser</a> closing or becoming minimized,
+          or due to an incoming call on a phone.
+        </p>
+        <p class="note">
+          User agents ultimately control what inaccesible means in this context, but are expected
+          to only fire mute and unmute events for interruptions that have external reasons.
```

Comment:
I do not really understand what 'interruptions that have external reasons' mean so I am not sure how this helps.
The wording does seem to exclude valid cases like:
- A user interacting with User Agent UI to temporarily mute capture.
- A user agent muting a track because another page started to capture a screen.
- A user agent muting a track after a large timer, like 24 hours.

Reply:
I think a good separation would be:
* The **capturing document**, by which I mean the entity holding the track, can directly pause the capture via [enabled](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/enabled).
* Every other cause else should be reflected through [muted](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/muted). Regardless of it is directly/indirectly caused by the capturing document.

For distinguishing different mute-causes I [have suggested](https://github.com/w3c/mediacapture-region/issues/9#issuecomment-1022334881) this:
```webidl
enum MuteCasue {
  "user",
  "target_lost",
  "zero_pixel_frame",
  ...
};

partial interface MediaStreamTrack {
  readonly sequence<MuteCasue> mute_causes;
}
```